hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -45,6 +45,17 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
             </div>
 
             <div class="form-group">
+                <label for="subscriptionState" class="col-md-3 control-label">Initial state</label>
+                <div class="col-md-9">
+                    <select required class="form-control" id="subscriptionState" name="subscriptionState" ng-model="subscription.state">
+                        <option value="PENDING">PENDING</option>
```

Comment:
Well, this is actually the default state of subscription upon creation right now. It is then automatically changed to ACTIVE (asynchronously).